media/js/base/mozilla-client.js

```diff
@@ -217,6 +217,7 @@ if (typeof Mozilla === 'undefined') {
                 'accurate': accurate,
                 'version': version,
                 'channel': channel,
+                'distribution': distribution,
```

Comment:
We should also add this value to the `window.dataLayer.push()` property on line 229 below.